docs/api.md

```diff
@@ -300,11 +300,12 @@ respx.get("all://*.example.org/foo/")
 
 ### Content
 Matches request raw *content*, using [eq](#eq) as default lookup.
-> Key: `content`  
-> Lookups: [eq](#eq)
+> Key: `content`
```

Comment:
```suggestion
> Key: `content`  
```

Reply:
Ah, well-spotted. Coincidentally I removed that hook today from my editor as it is too aggressive.

Reply:
Ah, well-spotted. Coincidentally I removed that hook from my editor today; it is too aggressive.